### PR TITLE
Update mimoto-issuers-config.json

### PR DESCRIPTION
**Pull request by gopalaswin**
adding a new issue in the mimoto issuer config

### CODE REVIEW

**Review by maheswaras — APPROVED** [review]

File: mimoto-issuers-config.json [hunk 1]
```diff
@@ -430,6 +430,106 @@
...
+      "client_id": "${mimoto.oidc.mosipid.partner.clientid}",
+      "client_alias": "mpartner-default-test-mosipid",
+      ".well-known": "http://inji-config-server.config-server/config/mimoto/default/qa/mimoto-mosipid-identity-wellknown.json",
+      "redirect_uri": "io.mosip.residentapp.inji://oauthredirect",
+      "authorization_audience": "https://${mosip.esignet.mosipid.host}/v1/esignet/oauth/v2/token",
+      "token_endpoint": "https://${mosip.api.public.host}/v1/mimoto/get-token/Mosip",
+      "proxy_token_endpoint": "https://${mosip.esignet.mosipid.host}/v1/esignet/oauth/v2/token",
+      "qr_code_type": "OVPRequest",
+      "enabled": "true"
+    },
```
Comment (vijay151096) [inline mimoto-issuers-config.json:533]: i think there is a extra comma in the end.

Comment (vijay151096) [reply mimoto-issuers-config.json:533]: pls use any json formatter side to validate the json once
ref : https://jsonformatter.curiousconcept.com/

**Review by vijay151096 — APPROVED** [review]
lgtm

File: mimoto-issuers-config.json [hunk 2]
```diff
@@ -430,6 +430,106 @@
...
+      ".well-known": "http://inji-config-server.config-server/config/mimoto/default/qa/mimoto-mock-identity-wellknown.json",
+      "redirect_uri": "io.mosip.residentapp.inji://oauthredirect",
+      "authorization_audience": "https://${mosip.esignet.mock.host}/v1/esignet/oauth/v2/token",
+      "token_endpoint": "https://${mosip.api.public.host}/v1/mimoto/get-token/Mock1",
+      "proxy_token_endpoint": "https://${mosip.esignet.mock.host}/v1/esignet/oauth/v2/token",
+      "qr_code_type": "OVPRequest",
+      "enabled": "true"
+    },
+{
+      "credential_issuer": "Mosip",
```
Comment (vijay151096) [inline mimoto-issuers-config.json:460]: Mosip Issuer is already avl in the config

File: mimoto-issuers-config.json [hunk 3]
```diff
@@ -430,6 +430,106 @@
...
+          "language": "fil"
+        }
+      ],
+      "protocol": "OpenId4VCI",
+      "client_id": "${mimoto.oidc.mosipid.partner.clientid}",
+      "client_alias": "mpartner-default-test-mosipid",
+      ".well-known": "http://inji-config-server.config-server/config/mimoto/default/qa/mimoto-mosipid-identity-wellknown.json",
+      "redirect_uri": "io.mosip.residentapp.inji://oauthredirect",
+      "authorization_audience": "https://${mosip.esignet.mosipid.host}/v1/esignet/oauth/v2/token",
+      "token_endpoint": "https://${mosip.api.public.host}/v1/mimoto/get-token/Mosip",
```
Comment (vijay151096) [inline mimoto-issuers-config.json:529]: This should be https://${mosip.api.public.host}/v1/mimoto/get-token/Mosip1 and not /mosip

**Pull request merged**